README.md

```diff
@@ -132,6 +132,7 @@ The policy includes a common configuration that applies to all the strategies:
 | `evictSystemCriticalPods` | `false` | [Warning: Will evict Kubernetes system pods] allows eviction of pods with any priority, including system pods like kube-dns |
 | `ignorePvcPods` | `false` | set whether PVC pods should be evicted or ignored |
 | `maxNoOfPodsToEvictPerNode` | `nil` | maximum number of pods evicted from each node (summed through all strategies) |
+| `evictFailedBarePods` | `false` | allow eviction of pods without owner references and in failed phase |
```

Comment:
Mentions of the keyword "bare":
- https://kubernetes.io/docs/concepts/workloads/controllers/job/#bare-pods
- https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#bare-pods
- https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/#bare-pods

So it seems to be a commonly used term. Reading the field the first time I could not help myself to relate it to a baremetal.

Reply:
Yeah, that's where I got the variable name from. It seemed to me that it would be the convention to use `bare pods` rather than `standalone pods`. Do we have any concerns about the naming?

Reply:
I can not think of a more suitable name. Bare pods and `evictFailedBarePods` field are practical.